### PR TITLE
GA campaign

### PR DESCRIPTION
**Pull request by yashghelani**

### CODE REVIEW

**Review by NejcZdovc — COMMENTED** [review]
- compare-with-google-analytics` should be just jsx file and not folder.
- I don't see where we would use this in navigation, it's giving me 404

File: app/routes/ga.jsx [hunk 1]
```diff
@@ -342,7 +379,7 @@ export default function Campaign() {
 
         <div className='cta-box campaign-section'>
           <div className='cta-text'>Learn how to fix your analytics today</div>
-          <form method='post'>
+          <form id='form-bottom' method='post'>
```
Comment (NejcZdovc) [inline app/routes/ga.jsx:377]: do you need `onSubmit={submission}` on this one as well?

File: app/routes/ga.jsx [hunk 2]
```diff
@@ -41,23 +49,49 @@ export let action = async ({ request }) => {
 
   let headers = new Headers()
   headers.append('Content-Type', 'application/json')
-  headers.append('x-api-key', 'ZWBQ5E48ND3VTPB')
+  headers.append('x-api-key', `4J6OqWceq69vqTNRQPCLD44n`)
```
Comment (NejcZdovc) [inline app/routes/ga.jsx:52]: can we put this in env variable if submit happens on server side?

Comment (yashghelani) [reply app/routes/ga.jsx]: Have updated the code to use an env variable

**Review by NejcZdovc — COMMENTED** [review]
- scroll bar visible
image

- form doesn't work for me
image

**Review by NejcZdovc — APPROVED** [review]

**Comment by yashghelani** [comment]
Closes #113

**Pull request merged**